### ISSUE TITLE
False positive on interface:  "expected onespace after colon in parameter"

### ISSUE THREAD

**Issue by QuantumInformation** [blakeembrey/tslint-config-standard#24]
Not sure if I should raise here or https://github.com/palantir/tslint/issues

```js
export interface ICodeGenerator {
  description: string
  elementAdded: (id: string) => string
  elementRemoved: (id: string) => string
  clickHappened: (queryPath: string) => string
  inputTextEdited: (queryPath: string, newValue: string) => string
  selectChange: (queryPath: string, event: Event) => string
}
```
gives error:

![image](https://cloud.githubusercontent.com/assets/216566/24756238/f8c7071a-1ad4-11e7-9dfa-569dc71865d7.png)

Using tslint 5.0.0
